default-network-params.json

```diff
@@ -9,16 +9,17 @@
         "beacon_extra_params":    [],
         "el_extra_params":        [],
         "validator_extra_params": [],
-        "builder_network_params": null
-	}],
-	"network_params":{
-		"preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
-		"num_validator_keys_per_node": 64,
-		"network_id": "3151908",
-		"deposit_contract_address": "0x4242424242424242424242424242424242424242",
-		"seconds_per_slot": 12,
-		"genesis_delay": 120,
-		"capella_fork_epoch": 5
-	},
-	"global_client_log_level": "info"
+        "builder_network_params": null,
+        "count": 5
```

Comment:
I would not set the default count to 5. Lets leave that at 1.

Reply:
Oh fixed. Thanks!